Templates/MonoGame.Templates.CSharp/content/MonoGame.Application.Android.CSharp/.template.config/template.json

```diff
@@ -1,21 +1,15 @@
 {
     "author": "MonoGame Team",
     "classifications": [
-        "MonoGame"
+        "MonoGame", "Games", "Mobile", "Android"
     ],
     "name": "MonoGame Android Application",
-    "groupIdentity": "MonoGame.Application.Android",
     "identity": "MonoGame.Application.Android.CSharp",
     "shortName": "mgandroid",
     "tags": {
         "language": "C#",
         "type": "project"
     },
-    "primaryOutputs": [
```

Comment:
`primaryOutputs` is needed for VS for Mac extension to know which project/s to reference to the solution 😅

Reply:
Will restore them. I initially thought that they were responsible for VS2022 not showing the templates, but apparently it was about the categories (two templates with the exact same tags and categories can't coexist in VS2022, which is weird).

Reply:
I kind of missed it, but you also removed `groupIdentity`, this property is used for the ability to connect multiple templates with different programming langauge to the same template name. Even tho we don't have F# or VB.Net templates, it's still useful to keep it around if we ever decide to maintain them. This, is of course, if its not creating any trouble with VS 😄

Reply:
Do you mind if we restore ```groupIdentity``` when we rework the F# and VB templates? Your call. Otherwise I think we're good.

Reply:
Its probably safer to add it.